man/src/oxide-config.1.md

```diff
@@ -0,0 +1,267 @@
+
+## MOVEMENT
+
+Left
+: Moves to the left
+
+right
+: Moves to the right
+
+## LAYOUT
+
+VerticalStriped
+: windows vertically next to each other
+
+HorizontalStriped
+: windows horizontally underneath each other
+
+None
+: if no argument is provided, the next layout is chosen
+
+## WORKSPACE_ARGS
+
+Next
+: next initialized workspace with a higher index than the current workspace
+
+Previous
+: next initialized workspace with a lower index than the current workspace
+
+Next_free
+: next available workspace with a higher index than the current workspace which is not initialized
+
+Index
+: workspace with the given index
+
+# ITERATIONS
+
+The interation commands provide the possibillity to change between workspaces when given a iteration number as shown in the example down below.
+
+iter
+: iterates over given number in order to change
+
+# DEFAULT KEYBINDINGS
+
+Here is a short overview of the default keybindings
+
+Meta+Shift+e
+: quits the window manager
+
+Meta+Shift+r
+: restarts the window manager
+
+Meta+Shift+q
+: kills the current window
+
+h/l
+: direction keys (left/right)
+
+Meta+<direction>
+: changes the focus to the <direction> window
+
+Meta+Shift+<direction>
+: moves the window to the <direction>
+
+Meta+f
+: changes the current window to fullscreen
+
+Meta+u
+: switches to the next layout
+
+Meta+i
+: changes the layout to vertical
+
+Meta+Shift+i
+: changes to layout to horizontal
+
+Right/Left;
+: workspace navigation keys (next/previous)
+
+Meta+<workspace direction>
+: changes to the <workspace direction> workspace
+
+Meta+n
+: opens a new workspace
+
+Control+Meta+<workspace direction>
+: moves a window to the <workspace direction> workspace
+
+Control+Meta+n
+: opens a new workspace and moves the window to it
+
+Meta+Shift+<workspace direction>
+: moves the window to the <workspace direction> workspace and follows it
+
+Meta+Shift+n
+: creates a new workspace, moves the window to it and follows
+
+Control+Meta+Down
+: quits the workspace
+
+Meta+t
+: opens a new firefox window
+
+1/2/3/4/5/6/7/8/9;
+: workspace numbers
+
+Meta+<workspace number>
+: switches to workspace <workspace number>
+
+Control+Meta+<workspace number>
+: moves window to workspace <workspace number>
+
+Meta+Shift+<workspace number>
+: moves window to workspace <workspace number> and follows it
+
+# BORDERS
+
+border_witdh
+: sets the border witdh of windows in pixels
+
+border_color
+: sets the border color and has to be entered in hexadecimal
+
+border_focus_color
+: sets the border color for focused nbdows and has to be entered in hexadecimal
+
+gap
+: gap between windows in pixels
+
+# EXECUTE
+
+exc
+: onetime execution when the window manager starts
+
+exec_always
+: is executed during start of the window manager and also at each restart
+
+# EXAMPLES
+
+## KEYBINDINGS
+
+```yaml
+cmds:
+  - keys: ["M", "t"]
+    commands:
+      - command: Exec
+        args: "firefox"
```

Comment:
Idk if firefox is the best idea here, like @janschaible and @FelixSchladt said before i would also agree to switch on dmenu.